spec/armor_payments/api/resource_spec.rb

```diff
@@ -57,13 +61,13 @@ module ArmorPayments
 
       describe "#get" do
         it "queries the host for a specific resource, with approprate headers" do
-          Timecop.freeze(2014, 2, 22, 12, 0, 0) do
-            resource.connection.should_receive(:get).with({
+          Timecop.freeze(time) do
+            expect(resource.connection).to receive(:get).with({
               path: '/wibble/123/resource/456',
               headers: {
-                "X_ARMORPAYMENTS_APIKEY"            => "my-api-key",
-                "X_ARMORPAYMENTS_REQUESTTIMESTAMP"  => "2014-02-22T17:00:00Z",
-                "X_ARMORPAYMENTS_SIGNATURE"         => "48886620cfebb95ffd9ee351f4f68d4f103a8f4bdc0e3301f7ee709ec2cf3c19588ae1b67aa8ee38305de802651fb10093cf1af40f467ac936185d551a58a844"
+                "x-armorpayments-apikey"            => "my-api-key",
+                "x-armorpayments-requesttimestamp"  => "2014-02-22T12:00:00Z",
+                "x-armorpayments-signature"         => Digest::SHA512.hexdigest("#{api_secret}:GET:/wibble/123/resource/456:#{Time.now.utc.iso8601}")
```

Comment:
Regarding the previous headers, do you mean the timestamp would change between the the `headers: {}` hash with the `Time.now` statement and the `resource.get()`?